routes/web.php

```diff
@@ -114,6 +114,8 @@
 Route::post('/api/User::Save', [Administration\UserController::class, 'save'])->middleware('admin');
 Route::post('/api/User::Delete', [Administration\UserController::class, 'delete'])->middleware('admin');
 Route::post('/api/User::Create', [Administration\UserController::class, 'create'])->middleware('admin');
+Route::post('/api/User::UpdateEmail', [Administration\UserController::class, 'updateEmail'])->middleware('read');
+Route::post('/api/User::GetEmail', [Administration\UserController::class, 'getEmail'])->middleware('read');
```

Comment:
I don't think `read` is the correct middleware here... I would think it should be `login` for the second one, and as for the first one -- probably `login` as well? For the first one we should probably be checking if the account isn't restricted ("locked") inside `updateEmail` (or are you doing it already?).

Reply:
I haven't got a check if a user is locked, but I can add that to the command to avoid sending to locked accounts & to the notify action to avoid creating notifications for locked accounts.

Reply:
Ill change the middleware to `login` Users that have restricted accounts will still be able to receive the notification, so would still need access to update their email